hooks/boot/01-kafo-hook-extensions.rb

```diff
@@ -113,8 +113,23 @@ def log_and_say(level, message, do_say = true, do_log = true)
     Kafo::KafoConfigure.logger.send(level, message) if do_log
   end
 
-  def execute(command, do_say = true, do_log = true)
-    exit 1 unless execute_command(command, do_say, do_log)
+  def execute!(command, do_say = true, do_log = true)
+    stdout_stderr, status = execute_command(command, do_say, do_log)
```

Comment:
Would this make it clear that it isn't a `Process::Status` object?
```suggestion
    stdout_stderr, success = execute_command(command, do_say, do_log)
```

Reply:
You tell me, the current code is already clear to me :)

Reply:
@ekohl what's the final thought here?